planet_scope/max_ndvi/README.md

```diff
@@ -0,0 +1,39 @@
+---
+title: 'Max Multitemporal NDVI'
+parent: Planetscope
+grand_parent: Planet
+layout: script
+permalink: /planet_scope/max_ndvi/
+nav_exclude: true
```

Comment:
Since this is a script returning maximum ndvi, I would suggest preparing an EO Browser script (eob.js) which enables statistical function on EOB and a raw vlue script (raw.js) which returns raw values and can be used by our Requests Builder for download purpose.

Please see the [example](https://github.com/sentinel-hub/custom-scripts/blob/main/contribute/example-multiple-scripts/index.md?plain=1#L6).